src/modules/utils.js

```diff
@@ -286,7 +286,13 @@ define([
         return pluginName + dep;
       }
 
-      return pluginName + normalizePath([baseName(moduleId), dep].join('/'));
+      var moduleBaseName = baseName(moduleId);
```

Comment:
if `baseName(moduleId)` is "" then it gets added as an empty element to the array, causing the array to begin with `/` when it shouldn't. The generated variable name then doesn't match the one generated for dependencies because it has an additional `_` at the begining.